src/cli/mod.rs

```diff
@@ -102,6 +129,18 @@ struct LoadCli {
 
     #[clap(long, value_parser)]
     field: Option<String>,
+
+    /// Path for cached public parameters
+    #[clap(long, value_parser)]
+    public_param_path: Option<PathBuf>,
+
+    /// Path for cached proofs
+    #[clap(long, value_parser)]
+    proof_path: Option<PathBuf>,
+
+    /// Path for cached commitments
+    #[clap(long, value_parser)]
+    commit_path: Option<PathBuf>,
```

Comment:
Let's not add docstrings on these so it's clear that `clap` is not generating help info from here